package.json

```diff
@@ -1,66 +1,79 @@
 {
   "private": true,
-  "license": "MIT",
```

Comment:
Having the license field in our root is nice because GH will show it in the repo information.